openvdb_houdini/houdini/SOP_OpenVDB_Sample_Points.cc

```diff
@@ -265,6 +302,24 @@ newSopOperator(OP_OperatorTable* table)
             "A subset of the input VDBs to be processed"
             " (see [specifying volumes|/model/volumes#group])"));
 
+    // vdb points groups
+    parms.add(hutil::ParmFactory(PRM_STRING, "vdbpointsgroups", "VDB Points Groups")
+        .setHelpText("Specify subset of VDB points to sample."
+                     " This parameter has no effect if there are no input VDB point grids")
+        .setChoiceList(&hvdb::VDBPointsGroupMenuInput1));
+
+    // rename "vel" attribute to "v"
+    parms.add(hutil::ParmFactory(PRM_TOGGLE, "renamevel", "Rename Vel To V")
+        .setDefault(PRMoneDefaults)
+        .setHelpText("When toggled on, if the input VDB's name is vel, "
+            "rename the VDB points attribute to v"));
+
+    // warn if attempting to overwrite existing attribute
+    parms.add(hutil::ParmFactory(PRM_TOGGLE, "attributeexists", "Attribute Exists Warning")
+        .setDefault(PRMoneDefaults)
```

Comment:
In my opinion, having this default to off would be preferable. To sample into an attribute with a compression codec, you would have to add it before sampling, so that workflow would always create a warning on this node by default. Again, this one would be good if it's possible to support for Houdini points in a trivial way (if it doesn't already).